packages/@react-aria/searchfield/src/useSearchField.ts

```diff
@@ -63,15 +75,20 @@ export function useSearchField(
 
   let onClearButtonClick = () => {
     state.setValue('');
-    searchFieldRef.current.focus();
+    inputRef.current.focus();
   };
 
+  let {labelProps, inputProps} = useTextField({
+    ...props,
+    value: state.value,
+    onChange: state.setValue,
```

Comment:
?

Reply:
Not sure what you're questioning? 😉

Reply:
sorry, i think this was a note to myself while i was reviewing